src/screens/epoch/types.ts

```diff
@@ -0,0 +1,22 @@
+export type InflationType = {
+  validator: number;
+  foundation: number;
+  total: number;
+  epoch: number;
+}
+
+export type InflationGovernorType = {
+  initial: number;
+  terminal: number;
+  taper: number;
+  foundation: number;
+  foundationTerm: number;
+}
+
+export type EpochScheduleType = {
+  slotsPerEpoch: number;
+  leaderScheduleSlotOffset: number;
+  warmup: boolean;
+  firstNormalEpoch: number;
+  firstNormalSlot: number;
+}
```

Comment:
wait you have a type file..why didn't you use it in the `hooks`

Reply:
Updated.

Reply:
<img width="362" alt="image" src="https://user-images.githubusercontent.com/42913823/165121074-387b31e4-3378-4b5d-a6c0-502d96f9dcd7.png">

I want something more like

```
const [state, setState] = useState<EpochState>({...})
```

inside the types:

```
export type EpochState = {
  loading: boolean;
  exists: boolean;
  inflation: InflationType;
  governor: InflationGovernorType;
  schedule: EpochScheduleType;
}
```

Reply:
Updated.